src/App.js

```diff
@@ -9,6 +9,17 @@ export class App extends Component {
     movies: data,
   };
 
+  addMovies = (newMovies) => {
```

Comment:
Ти один фільм додаєш
```suggestion
  addMovie = (movie) => {
```